### PR TITLE
IA-related updates to SEO and metadata content

### PR DESCRIPTION
**Pull request by sarahemlin**
# Pull Request Description

I updated the Style Guide in a few places to include IA & SEO information I think is particularly important.

JIRA URL: <https://jira.mongodb.org/browse/DOCSP-33533>

Staging URL: <https://preview-mongodbsarahemlin.gatsbyjs.io/meta/DOCSP-33533/>

Build log: <https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6633fbeeb67dbf804a71e513>

### CODE REVIEW

File: source/style-guide/seo-guidelines.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@ Keywords
 --------
 
 You can add the actual terms that users enter into search engines 
-(keywords) to the body of your page to improve its SEO.
+(keywords) to the body of your page to improve its SEO.  Google Search Console provides details about the keywords our users enter and can be requested from the IA and SEO teams.
```
Comment (corryroot) [inline source/style-guide/seo-guidelines.txt:24]: Is there a process for IA and SEO info requests? Something we can link to?

```suggestion
(keywords) to the body of your page to improve its SEO.  Google Search Console provides details about the keywords our users enter and you can request this information from the IA and SEO teams.  
```

**Review by corryroot — APPROVED** [review]
LGTM! I commented with a nit and question.

File: source/style-guide/seo-guidelines.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@ Search Engine Optimization Guidelines
 
 Search Engine Optimization (SEO) considers the actual terms that users 
 enter into search engines (keywords) and employs best practices to 
-improve traffic to web pages from search engines. The following 
+improve traffic to web pages from search engines.  Close attention to page metadata and keywords in the page content can improve placement on search results pages.  The following
```
Comment (lindseymoore) [inline source/style-guide/seo-guidelines.txt:9]:
```suggestion
improve traffic to web pages from search engines.  Close attention to page metadata and keywords in the page content can improve page placement in search results.  The following 
```
Suggestion to make more clear!

File: source/style-guide/seo-guidelines.txt [hunk 3]
```diff
@@ -91,7 +91,7 @@ Descriptions
 ------------
 
 The description is a snippet that appears under the link in the search 
-results, which can impact SEO. Write these descriptions according to 
+results, which is essential for SEO. Write these descriptions according to
```
Comment (lindseymoore) [inline source/style-guide/seo-guidelines.txt:94]:
```suggestion
results and is essential for SEO. Write these descriptions according to 
```
Suggestion

**Review by lindseymoore — APPROVED** [review]
Small nits and would love an answer to Cory's question included in the section if there is an official process! Otherwise, LGTM!

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM! Thanks for these updates @sarahemlin

**Comment by sarahemlin** [comment]
@corryroot & @lindseymoore, you just need to ask me for Search Console data. If I'm not around or it's too complicated, I turn to SEO. I didn't figure we wanted to put "Email Sarah Lin" in the style guide, but if we do, that's cool!

If one of you would please merge, I would appreciate it!

**Pull request merged**